cypress/integration/team-members/profile.js

```diff
@@ -0,0 +1,108 @@
+describe('Team members profile info', () => {
+  it('Should not have permissions when not logged in', () => {
+    cy.visit('http://localhost:3000/team-members/profile/admin_member6@gmail.com')
+
```

Comment:
we should also check that things specific to a team meber don't exist on the page